index.html

```diff
@@ -0,0 +1,174 @@
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300&display=swap"
+			rel="stylesheet"
+		/>
+		<script
+			src="https://kit.fontawesome.com/6c59d03160.js"
+			crossorigin="anonymous"
+		></script>
+		<title>Portfolio</title>
+	</head>
+	<body>
+		<section>
+			<div class="card">
+				<img src="images/Portfolio-1.png" alt="" />
+				<h1>Facebook 360</h1>
+				<ul class="description">
+					<li><h3>FACEBOOK</h3></li>
+					<li><i class="fa-solid fa-circle"></i> Full Stack Dev</li>
+					<li><i class="fa-solid fa-circle"></i> 2015</li>
+				</ul>
+				<p>
+					Exploring the future media in Facebook's first Virtual
+					reality app; a place to discover and enjoy 360 photos and
+					videos on Gear VR.
+				</p>
+				<ul class="language-used">
+					<li>html</li>
+					<li>css</li>
+					<li>javaScript</li>
+				</ul>
+				<button type="submit">See Project</button>
+			</div>
+
+			<div class="card">
+				<img src="images/Portfolio-2.png" alt="" />
+				<h1>Multi-post stories</h1>
+				<ul class="description">
+					<li><h3>FACEBOOK</h3></li>
+					<li><i class="fa-solid fa-circle"></i> Full Stack Dev</li>
+					<li><i class="fa-solid fa-circle"></i> 2015</li>
+				</ul>
+				<p>
+					Experimental content creation feature that allows users to
+					add to an existing story over the course of a day without
+					spamming without their friends.
+				</p>
+				<ul class="language-used">
+					<li>html</li>
+					<li>css</li>
+					<li>javaScript</li>
+				</ul>
+				<button type="submit">See Project</button>
+			</div>
+
+			<div class="card">
+				<img src="images/Portfolio-3.png" alt="" />
+				<h1>Tonic</h1>
+				<ul class="description">
+					<li><h3>CANOPY</h3></li>
+					<li><i class="fa-solid fa-circle"></i> Back End Dev</li>
+					<li><i class="fa-solid fa-circle"></i> 2018</li>
+				</ul>
+				<p>
+					A daily selection of privately personalized reads; no
+					accounts or sign-ups required.
+				</p>
+				<ul class="language-used">
+					<li>html</li>
+					<li>css</li>
+					<li>javaScript</li>
+				</ul>
+				<button type="submit">See Project</button>
+			</div>
+
+			<div class="card">
+				<img src="images/Portfolio-4.png" alt="" />
+				<h1>Uber Navigation</h1>
+				<ul class="description">
+					<li><h3>Uber</h3></li>
+					<li><i class="fa-solid fa-circle"></i> Lead developer</li>
+					<li><i class="fa-solid fa-circle"></i> 2018</li>
+				</ul>
+				<p>
+					A smart assistant to make driving more safe, efficient, and
+					fun by unlocking your most expensive computer: your car.
+				</p>
+				<ul class="language-used">
+					<li>html</li>
+					<li>css</li>
+					<li>javaScript</li>
+				</ul>
+				<button type="submit">See Project</button>
+			</div>
+		</section>
+
+		<section>
+			<div class="about">
+				<h1>About Myself</h1>
```

Comment:
- [ ] To make your HTML more friendly to assistive technologies and SEO, it is nice to have a single `<h1>` tag which wraps main heading of the page, please change the above sub heading to an `<h2>` or any header tag different from `<h1>`.